app/collections/lane/[slug]/page.tsx

```diff
@@ -19,6 +19,7 @@ export async function generateMetadata({
 }
 
 export default function Lane({ params }: LaneProps) {
+  const pageName = `dc|collections|lane|${params.slug}`;
```

Comment:
Remove `dc|` from here. The prefix is added automatically in the `trackVirtualPageView` function when it gets sent to Adobe.